lms/djangoapps/teams/static/teams/js/views/team_discussion.js

```diff
@@ -13,7 +13,7 @@
                 render: function() {
                     var discussionModuleView = new DiscussionModuleView({
                         el: this.$el,
-                        readOnly: this.$el.data('read-only'),
+                        readOnly: this.$el.data('read-only') === true,
```

Comment:
You shouldn't need ``=== true``. If it is already a boolean, then the test does nothing. If it isn't already a boolean, then it will always return false.